client/src/components/Results/Cards.js

```diff
@@ -69,9 +69,9 @@ class Card extends React.Component {
       parsedListing: record.listing,
       parsedPhone: cardPhoneTextFilter(record),
       parsedWeb: cardWebAddressFixer(record.website),
-      parsedStreet: `${cardTextFilter(record.street)} ${cardTextFilter(
+      parsedStreet: record.street !== null && record.street !== '' ? `${cardTextFilter(record.street)} ${cardTextFilter(
```

Comment:
Are the db string results different from before?

Reply:
Yes